junit-jupiter-engine/src/test/java/org/junit/jupiter/engine/extension/OrderedMethodTests.java

```diff
@@ -123,34 +124,61 @@ private void assertOrderAnnotationSupport(Class<?> testClass) {
 
 	@Test
 	void random() {
+		var tests = executeTestsInParallel(RandomTestCase.class);
+
+		tests.assertStatistics(stats -> stats.succeeded(callSequence.size()));
+
+		assertThat(threadNames).hasSize(1);
+	}
+
+	@Test
+	@TrackLogRecords
+	void randomWithBogusSeedRepeatedly(LogRecordListener listener) {
+		String seed = "explode";
+		String expectedMessage = "Failed to convert configuration parameter [" + Random.RANDOM_SEED_PROPERTY_NAME
+				+ "] with value [" + seed + "] to a long. Using System.nanoTime() as fallback.";
+
 		Set<String> uniqueSequences = new HashSet<>();
 
 		for (int i = 0; i < 10; i++) {
 			callSequence.clear();
+			listener.clear();
 
-			var tests = executeTestsInParallel(RandomTestCase.class);
+			var tests = executeTestsInParallelWithRandomSeed(RandomTestCase.class, seed);
 
 			tests.assertStatistics(stats -> stats.succeeded(callSequence.size()));
 
 			uniqueSequences.add(callSequence.stream().collect(Collectors.joining(",")));
+
+			// @formatter:off
+			assertTrue(listener.stream(Random.class, Level.WARNING)
+				.map(LogRecord::getMessage)
+				.anyMatch(expectedMessage::equals));
+			// @formatter:on
 		}
 
-		// We assume that at least 3 out of 10 are different...
-		assertThat(uniqueSequences).size().isGreaterThanOrEqualTo(3);
-		// and that at least 2 different threads were used...
-		assertThat(threadNames).size().isGreaterThanOrEqualTo(2);
+		assertThat(uniqueSequences).size().isEqualTo(1);
 	}
 
 	@Test
 	@TrackLogRecords
-	void randomWithBogusSeed(LogRecordListener listener) {
-		String seed = "explode";
-		String expectedMessage = "Failed to convert configuration parameter [" + Random.RANDOM_SEED_PROPERTY_NAME
-				+ "] with value [" + seed + "] to a long. Using System.nanoTime() as fallback.";
+	@Disabled("needs to be loaded within own Classloader, not sure how to verify")
```

Comment:
i am not sure, how i can properly test this. i am not sure how to use a custom class loader in combination with @trackLogRecords. Guidance is highly appreciated

Reply:
If you cannot get the testing of the logging for the static initialization to work, don't worry about it. We can always add a test for that later after merging the PR.

Reply:
:+1: